src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,21 +143,26 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
-    let dropdownItems = [
-      {
-        className: 'hidden',
-        html: 'More',
-        id: JobActionItem.MORE
-      },
-      {
+    let job = MetronomeStore.getJob(this.props.params.id);
+    let dropdownItems = [];
```

Comment:
@kennyt I actually asked asked @pierlo-upitup to change this to array `push`  in https://github.com/dcos/dcos-ui/pull/629#discussion_r69585121, as using `splice` would mean that whenever we want to change the order or need to add another button we would have to also adjust the splice call which could result in errors.